### DISCUSSION TITLE
openvscode-releases workflows question

### DISCUSSION THREAD

**Discussion by Galcarmi** [gitpod-io/openvscode-server#433] [Q&A]
Hey guys,

https://github.com/gitpod-io/openvscode-releases
can you please help me understand what are the main differences between insiders.yml flow and the release.yml flow?
what is the purpose of insiders.yml?

thanks !

**Comment by jeanp413** [comment]
@Galcarmi `insiders.yml` rebases this repo `main` branch with `upstream/main` branch so we have the latest changes daily. `release.yml` does the actual release i.e. github and docker release
